### DISCUSSION TITLE
Error 1 while parsing

### DISCUSSION THREAD

**Discussion by pakiec22** [TumblThreeApp/TumblThree#300] [General]
The error happens almost immediately when trying to crawl:
https://www.tumblr.com/blog/view/[...]

Here is the error 
https://imgur.com/a/Xx6ND[...]

**Comment by thomas694** [comment]
Thanks for reporting a problem. We'll have a look.

**Comment by thomas694** [comment]
> **Error:**
> A post from '...' was not parsable. Setting 'posts per page' in the Details to a lower value might increase the downloadable content.
> **Internal error description**
> There was an error deserializing the object of type TumblThree.Applications.DataModels.TumblrSvcJson.TumblrJson. Encountered unexpected character '<'.

It looks like the most probable reason is that you haven't logged in yet. Go to Settings -> Connection, log in and accept the privacy consent.

**Comment by pakiec22** [comment]
Probably that was the case, anyway updated the app, logged in again and now is fine by now
